docs/notifications/notifications.md

```diff
@@ -6,15 +6,48 @@ sidebar_position: 1
 
 # Notifications
 
-Moonguard ships with some notifications for different use cases and scenarios, we've added the following notifications:
+MoonGuard ships with some notifications for multiple use cases and scenarios, the following notifications are available:
 
-- New Exception Log Group.
 - Request Took Longer Than Maximum Duration.
 - SSL Certificate Check Failed.
 - SSL Certificate Expires Soon.
 - Uptime Check Failed.
 - Uptime Check Recovered.
+- New Exception Log Group.
+
+MoonGuard uses two channels to send notifications by default: email and slack.
+This can be modified by your own preference in the configuration file:
+
+```bash
+[
+  //...
+  'notifications' => [
+    'channels' => ['mail', 'slack'],
+    'slack' => [
+      'webhook_url' => env('SLACK_WEBHOOK_URL'),
+    ],
+  ],
+];
+```
```

Comment:
This changed on the latest version of MoonGuard, the default channel for Notifications is `mail` we need to mention that you can add the `slack` channel, redirect the user to the Slack section if they need to setup slack notifications.